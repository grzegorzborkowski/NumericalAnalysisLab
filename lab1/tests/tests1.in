float
-1 -1 1 -1
2

1 1 1 1
1 0.5 0.333333 0.25
0.5 0.333333 0.25 0.2
0.333333 0.25 0.2 0.166667

# float -> precyzja
# potem wektor X
# potem norma wektora X
# potem macierz