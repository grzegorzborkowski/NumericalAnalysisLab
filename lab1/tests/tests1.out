-1 -0.999981 0.999961 -0.999978
1.99996


#wektor X obliczony metoda Gaussa
#norma wektora X